### ISSUE TITLE
AI hate my roads

### ISSUE THREAD

**Issue by TechpriestEnginseer** [LoneGazebo/Community-Patch-DLL#4964]
Capitulate someone. 

Use my worker to build roads to their city for easier access should they rebel and trade convenience.

They start tearing up the roads.

>:(

**Comment by ilteroi** [comment]
iirc, ai will remove any roads which aren't useful. meaning either economically (city connections) or strategically (citadel connections).

**Comment by LoneGazebo** [comment]
Turn on ‘automated workers won’t remove improvements’

**Comment by ilteroi** [comment]
that setting only affects human workers, doesn't it?

**Comment by LoneGazebo** [comment]
Yes. But isn’t that the issue?

**Comment by TechpriestEnginseer** [comment]
no, I'd like to connect my roads through vassal cities, but vassals are tearing it up

**Comment by LoneGazebo** [comment]
Ah. Well it’s their land. ;)

**Comment by TechpriestEnginseer** [comment]
im the master tho :(

**Comment by LoneGazebo** [comment]
master's roads will now be ignored.

**Comment by TechpriestEnginseer** [comment]
master's roads are not ignored in current version.
